Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -455,12 +467,6 @@ - (AVCaptureSession *)newSessionWithCaptureDevice:(AVCaptureDevice *)captureDevi
     return newSession;
 }
 
-- (AVCaptureDeviceInput *)deviceInputForCaptureDevice:(AVCaptureDevice *)captureDevice error:(NSError **)error {
```

Comment:
To add context to why this was the case: the `AVCaptureDeviceInput` instance is created in a few different spots in MTBBarcodeScanner. If we were configuring this object in any way, it _does_ make sense to extract that into it's own method, but we're currently just returning it as-is. So, this change is fine.